helm/sealed-secrets/values.yaml

```
@@ -324,6 +324,18 @@ networkPolicy:
   ## @param networkPolicy.enabled Specifies whether a NetworkPolicy should be created
   ##
   enabled: false
+  ## NetworkPolicy Egress configuration
+  ##
+  egress:
+    ## @param networkPolicy.egress.enabled Specifies wheter a egress is set in the NetworkPolicy
+    ##
+    enabled: false
+    ## @param networkPolicy.egress.kubeapiCidr Specifies the kubeapiCidr, which is the only egress allowed. If not set, kubeapiCidr will be found using Helm lookup
+    ##
+    kubeapiCidr: 
```

Looks amazing the PR. Just minor request, could you set up with empty quotes, please?

```suggestion
    kubeapiCidr: ""
```